Valentin_Yakimenko/Module 3 - Exceptions and IO streams/HomeTaskThree/src/main/java/ua/valentin/propert/main/Config.java

```
@@ -0,0 +1,46 @@
+package ua.valentin.propert.main;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * Created by valentin.yakimenko on 15.03.16.
+ */
+public class Config {
+    private static Config instance = null;
+    private Properties properties = null;
+
+    private Config(String nameOfProperties) throws PropertiesFileNotFoundException {
+        properties = new Properties();
+        try (FileInputStream fis =
+                     new FileInputStream(new File(nameOfProperties))) {
+            properties.load(fis);
+        } catch (FileNotFoundException e) {
+            throw new PropertiesFileNotFoundException
+                    ("File: " + nameOfProperties + ", not found");
+        } catch (IOException e) {
+            //catch here
```

consider some kind of handling. don't leave am empty catch here and in other classes, because in this case code that invoke your method/constructor will not be notified about the exception.